Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Utilize the Node.js 14 official image
+FROM node:18-alpine

```

Comment:
What is the current latest version?

Reply:
Current version in v20.17.0 using this one because in our volta package 18 version is pinned